docs/internals/sorting.md

```diff
@@ -0,0 +1,93 @@
+# Sorting
+
+Quickwit can sort results based on fastfield values or score. This document discuss where and ho
+ it happens.
+It also tries to describe optimizations that may be enabled (but are not necessarily implemente)
+by this behavior.
+
+# Current state
+(this section shall be removed once proposed changes are accepted)
+
+## Behavior
+
+Sorting is controlled by the `sort_by` query parameter. It accepts a comma separated list of fields
+to use for sorting. Sorting is Ascending by default. The sorting order can be reversed by prefixing
```

Comment:
Crazy idea:

Should we want descending by default?

In elasticsearch it is descending by default for _score, ascending for everything else.
For timestamp, descending is better too, so maybe descending by default is a good idea.

Reply:
Aaaaaaah I had missed this was a description of the current state!